.github/workflows/linting.yml

```diff
@@ -20,3 +20,8 @@ jobs:
           python-version: ${{ inputs.python }}
       - run: pip install -r requirements-dev.txt
       - run: flake8 .
+  linting-black:
+    runs-on: ${{ inputs.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: psf/black@stable
```

Comment:
do we want to use the latest always, or do we want to pin the version? 

This should be the same as what we have in `pre-commit` hooks, which right now is:

```
-   repo: https://github.com/psf/black
    rev: 22.3.0
    hooks:
    -   id: black
```

Reply:
Good point; let's discuss in the community call, since I think that we should bump `black` up to v23, which appears to resolve some of the arguments between `black` and `flake8`.

Reply:
I might not make it to the call today, but I'm in favor of upgrading black here.

Reply:
@fairlearn/fairlearn-maintainers , since we didn't really have a quorum at the community call, do you have any objections to bumping up the version of `black` to 23?

Reply:
Have bumped.... will wait until after the community call, but at this point I'm leaning towards merging _unless_ there's an objection.